taxonomies/ingredients_processing.txt

```diff
@@ -2125,18 +2345,32 @@ nl: ongezouten
 pl: bez soli, niesolone
 pt: sem sal, não salgado
 
-en: sweetened
-bg: подсладено, подсладен, подсладена, подсладени
-de: gesüßt, gesüßte
-es: endulzado, endulzada, endulzados, endulzadas
-fi: makeutettu, makeutetut
-hr: zaslađene
-hu: édesített
-it: addolcito, addolciti, addolcita, addolcite
-nl: gezoet, gezoete
+en: sweetened, sugared, with sugar
+bg: подсладено, подсладен, подсладена, подсладени, захаросано, захаросан, захаросана, захаросани
+ca: ensucrada, amb sucre, dolc de
+cs: slazené
+da: sockrad, sukret
```

Comment:
“sockrad” is Swedish. “sødet” or maybe “forsødet” are direct translations of “sweetened”. Not sure which is more common.
```suggestion
da: sødet, sukret
```